arch/arm/configs/tsimx6ul_defconfig

```diff
@@ -857,26 +714,19 @@ CONFIG_DRM_IMX=y
 CONFIG_DRM_IMX_PARALLEL_DISPLAY=y
 CONFIG_DRM_MXSFB=y
 CONFIG_FB_MODE_HELPERS=y
+# CONFIG_FB_MX3 is not set
 CONFIG_FB_ST7565P=m
-CONFIG_DRM_FBDEV_EMULATION=y
```

Comment:
We definitely need this. This actually should probably be added to all configs, but for sure would be needed on 6ul with display support.

Reply:
Added to requirements, surprised this one wasn't already there.